README.md

```diff
@@ -81,35 +72,24 @@ python google_test_runner.py
 ## Command line options
 
 ```python
-python custom_runner.py --env [browserstack_environments] --tests [tests_to_run] --browserstack [username:token]
-
---env direct # runs tests on browserstack combinations
-             # passed to script through BROWSERSTACK environment variable (json)
+--platform web         # define platform on which run tests (currently supported: web, mobile, generic)
+--environmnet local    # define environment in which run tests (currently supported: local, browserstack, appium)
```

Comment:
please add `remote` as well (since `remote_module` branch was already merged in)